test/annotation_utils.t

```diff
@@ -864,6 +873,170 @@ subtest '_run_rast_workflow_ann' => sub {
 };
 
 
+## Just print to explore the finalgenome data structure and find the contig_ids##
```

Comment:
If you know the output from a certain input, use the tests to make sure that it is what you expect. Dumping a load of data structures to STDOUT can obscure test failures and does not help anyone who is unfamiliar with the codebase from diagnosing problems. Every time you've put a print statement below, you should be thinking about replacing it with known values and doing a `cmp_deeply` or similar comparison. If you want debugging of test failures, you can use `diag explain` to show where the structures differ, e.g.

```perl

cmp_deeply $got, $expected, 'running method generated correct results'
  or diag explain {
    got    => $got,
    expect => $expected,
  };
```

Reply:
Before I learned what the data structures looked like, these blocks of printouts served a good mission.  Now I am removing them because they were for me the developer to figure things out.